tokio-timer/src/delay_queue.rs

```diff
@@ -501,10 +501,13 @@ impl<T> DelayQueue<T> {
 
         self.slab[key.index].when = when;
 
+        let next_poll = self.wheel.poll_at()
+            .map(|t| self.start + Duration::from_millis(t));
+
```

Comment:
Would it make sense for this to be inside the if statement below?

Reply:
@ralith I had also thought about that, but as written, it didn't work as `self` couldn't be immutably borrowed by the closure. It occurred to me this morning that we could just assign `self.start` to a variable and move that into the closure, so I've changed this. 👍